dlrover/python/brain/hpsearch/bo.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import List, Tuple
+
+import torch
+from botorch.acquisition import qExpectedImprovement, qNoisyExpectedImprovement
+from botorch.fit import fit_gpytorch_mll
+from botorch.models import FixedNoiseGP, SingleTaskGP
+from botorch.models.transforms.input import Normalize
+from botorch.models.transforms.outcome import Standardize
+from botorch.optim import optimize_acqf
+from botorch.sampling.normal import SobolQMCNormalSampler
+from gpytorch.mlls import ExactMarginalLogLikelihood
+
+from dlrover.python.brain.hpsearch.base import OptimizerBase, RunResult
+
+
+class BayesianOptimizer(OptimizerBase):
```

Comment:
Add the docstring to introduce the `BayesianOptimizer` and  an example to show how to use the `BayesianOptimizer`.

Reply:
Added.